lib/NUPACK/emscripten/FullEnsemble.cpp

```diff
@@ -18,6 +18,9 @@ extern DBL_TYPE *pairPrPb;  //for pseudoknots
 extern double CUTOFF;
 extern int Multistranded;
 
+bool forEternaDigest = true;
+bool notforEternaDigest= false;
+
```

Comment:
Unused?

Reply:
This is used for the newdotparens stuff and will be taken out when I pull that function out.

Reply:
removed